use-timescale/metrics-logging/aws-cloudwatch.md

```diff
@@ -22,6 +23,8 @@ This page shows you how to create an Amazon CloudWatch exporter in $CONSOLE, and
 
 <PrereqsCloud />
 
+<NotSupportedAzure />
```

Comment:
Is it on the roadmap?

Reply:
I don't know. The partial is the same for everything.